bindings/python/docs/source/schemas.rst

```diff
@@ -0,0 +1,138 @@
+.. _schema usage:
+
+PyMongoArrow Schemas and Their Usage
+====================================
+
+While there are many similarities between `MongoDB Projections <https://www.mongodb.com/docs/manual/reference/operator/projection/positional/>`_
+and :class:`pymongoarrow.api.Schema`'s, there are a few important differences.
+Schemas do not support `"dotted" notation <https://www.mongodb.com/docs/manual/core/document/#dot-notation>`_, instead
+you must either specify a nested schema, or flatten your data using projections.
+
+
+Nested Data With Schema
+-----------------------
+
+With aggregate or find methods, you can provide a schema using the struct object. Note that there can be conflicting
+names in sub-documents compared to their parent documents.
+
+.. code-block:: python
+
+   >>> from pymongo import MongoClient
+   >>> from pymongoarrow.api import Schema, find_arrow_all
+   >>> from pyarrow import struct, field, int32
+   >>> coll = MongoClient().db.coll
+   >>> coll.insert_many([{
+   >>>     "start": "string",
+   >>>     "prop": {
+   >>>         "name": "foo",
+   >>>         "start": 0,
+   >>>     }
+   >>>   }, {
+   >>>       "start": "string",
+   >>>       "prop": {
+   >>>         "name": "bar",
+   >>>         "start": 10,
+   >>>     }
+   >>>   },])
+   >>> df=find_arrow_all(coll,{}, schema=Schema({"start": str, "prop": struct([field("start", int32())])}))
```

Comment:
Please use consistent code style. Above we use `coll = MongoClient` and here we use `df=find_arrow_all(coll,{},...` without spaces.

Reply:
`df` -> `arrow_table` which is what we call the find_arrow_all result elsewhere in the docs.

Reply:
Done.